src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,29 @@
+package seedu.addressbook.data.tag;
+import seedu.addressbook.data.person.Person;
+
+/**
+ * Each Tagging object will represent an adding or deleting 
+ * of a tag for a specific person that happened during that session
+ * Guarantees: immutable
+ */
+public class Tagging {
+
+	private Person person;
+	private Tag tag;
+	private String tagAction;
```

Comment:
think why `String` may not be the best data type for this particular field?
what would be more appropriate?